README.md

```diff
@@ -6,14 +6,19 @@ A simple full stack cloud app you can reference to build your own cloud ready ap
 
 ---
 
+## For step-by-step on how to run and deploy application to the cloud follow the follow the following link:
```

Comment:
Wording suggestion: 
Step-by-step on how to deploy the `hello_cloud_demo` app with IBM Cloud, Docker and Kubernetes: